plugins/net_plugin/net_plugin.cpp

```diff
@@ -3694,11 +3674,9 @@ namespace eosio {
          blk_send_branch( msg.req_blocks.ids.empty() ? block_id_type() : msg.req_blocks.ids.back() );
          break;
       case normal :
-         peer_dlog( this, "received request_message:normal" );
-         if( !msg.req_blocks.ids.empty() ) {
-            blk_send( msg.req_blocks.ids.back() );
-         }
-         break;
+         peer_wlog( this, "Invalid request_message, req_blocks.mode = normal" );
```

Comment:
Is `normal` not supported any more?

Reply:
Has not been for a very long time. I checked v2.0.0 of eosio/eos and it was not used even in that version. It was used in v1.0.0. I didn't track it down any further.

Reply:
Tests are failing, so I've missed something.

Reply:
Fixed